cspell.json

```diff
@@ -84,6 +84,7 @@
         "POIs",
         "Qbel",
         "Rasm",
+        "Raycast",
```

Comment:
What is this for?

Reply:
It's basically a dictionary of words that the spelling extension should ignore. So certain words from the API that are correct can be added it.